local-cli/link/windows/patches/makePackagePatch.js

```diff
@@ -0,0 +1,10 @@
+const applyParams = require('./applyParams');
+
+module.exports = function makePackagePatch(packageInstance, params, prefix) {
+  const processedInstance = applyParams(packageInstance, params, prefix);
+
+  return {
+    pattern: 'new MainReactPackage()',
```

Comment:
Is it possible to capture regex groups and re-apply them in the patch?  If so, you could capture the whitespace for the `(\s*)new MainReactPackage()` and apply it to the patch so there's matching whitespace.  Thinking something like:
```
{
    pattern: '(\s*)new MainReactPackage()',
    patch: ",\n$1' + processedInstance
}
```